crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -230,6 +262,18 @@ where
         self.into_configs().after(set)
     }
 
+    /// Run before all systems in `set` and systems in `set` do not care about the `deferred` effects
+    /// being applied.
+    fn before_ignore_deferred<M>(self, set: impl IntoSystemSet<M>) -> SystemConfigs {
+        self.into_configs().before_ignore_deferred(set)
+    }
+
+    /// Run after all systems in `set` and the deferred effects of systems in `set` do not need to be a
+    /// applied before running.
```

Comment:
```suggestion
    /// Run after all systems in `set`. Unlike [`after`](Self::after), this will not wait for the deferred effects of systems in `set` to be applied.
```